Apps/W1/HybridGP/app/src/pages/GPMigrationSettingsList.page.al

```diff
@@ -24,152 +24,7 @@ page 4021 "GP Migration Settings List"
                     ToolTip = 'Name of the company';
                     Width = 6;
                 }
-                field("Global Dimension 1"; Rec."Global Dimension 1")
-                {
-                    ApplicationArea = Basic, Suite;
```

Comment:
You cannot delete the fields, it will be seen as a breaking change. You should set the visible to false and Obsolete Pending. There should also be tags added with #if Clean22 --code #endif

Reply:
Updated